### PR TITLE
change functionality for sending email

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: app/maildelay/settings.py [hunk 1]
```diff
@@ -202,10 +202,10 @@
...
 
-EMAIL_HOST_USER = 'maildelay@maildelay.ml'
-DEFAULT_FROM_EMAIL = 'maildelay@maildelay.ml'
-EMAIL_HOST_PASSWORD = '3p7KDn4FugQQ'
-EMAIL_HOST = 'maildelay.ml'
+EMAIL_HOST_USER = 'maildelay@dev.rmd.io'
+DEFAULT_FROM_EMAIL = 'maildelay@dev.rmd.io'
+EMAIL_HOST_PASSWORD = 'KOhxgU8qHEtj'
+EMAIL_HOST = 'smtp.syhosting.ch'
 EMAIL_FOLDER = 'INBOX'
```
Comment (winged) [inline app/maildelay/settings.py]: Please don't put working credentials in the source, even if it's only for the dev env.

Change those credentials, and put some placeholder in here instead

File: app/mails/management/commands/sendmail.py [hunk 2]
```diff
@@ -1,6 +1,4 @@
-import email
-import smtplib
-import logging
+import email, smtplib, logging, traceback, sys
```
Comment (winged) [inline app/mails/management/commands/sendmail.py]: Style: Don't import multiple modules at once. Also, `traceback` and `sys` are not used anywhre

File: app/maildelay/settings.py (hunk 1 above)
Comment (JohnL17) [inline app/maildelay/settings.py]: I rebased the history and put some placholder in.

File: app/mails/management/commands/sendmail.py (hunk 2 above)
Comment (JohnL17) [inline app/mails/management/commands/sendmail.py]: I changed that.

**Comment by winged** [comment]
Please let me know when the logins are fixed, then we can merge this.

**Comment by JohnL17** [comment]
The login is fixed. I think we can merge it.

**Pull request merged**